Gemfile.lock

```diff
@@ -330,6 +330,7 @@ PLATFORMS
   arm64-darwin-22
   arm64-darwin-23
   x86_64-darwin-20
+  x86_64-darwin-23
```

Comment:
It seems that no one in our team uses macOS 14 with an Intel-based processor. Could you make this change in a separate commit?

Reply:
Thanks, done